orchagent/port.h

```diff
@@ -11,7 +11,7 @@ extern "C" {
 #include <map>
 
 #define DEFAULT_PORT_VLAN_ID    1
-#define DEFAULT_MTU              9100
+#define DEFAULT_MTU             1492
```

Comment:
>DEFAULT_MTU [](start = 8, length = 11)

Also fix DEFAULT_MTU_STR if has the same bug?

Reply:
that's not the same issue here i think. those are used to ensure configurations set to 9100.

Reply:
Yes, DEFAULT_MTU_STR is not the same as this.